wee_slack.py

```diff
@@ -637,6 +637,19 @@ def buffer_input_callback(signal, buffer_ptr, data):
     return w.WEECHAT_RC_OK
 
 
+# Workaround for supporting multiline messages. It intercepts before the input
+# callback is called, as this is called with the whole message, while it is
+# normally split on newline before being sent to buffer_input_callback
+def input_text_for_buffer_cb(data, modifier, current_buffer, string):
+    if current_buffer not in EVENTROUTER.weechat_controller.buffers:
+        return string
+    message = decode_from_utf8(string)
+    if "\n" in message and not message.startswith("/"):
```

Comment:
Just a really minor nitpick: the `message.startswith('/')` check should've probably gone first, since that's a constant-time check. Not that we _really_ care much about performance anyway.

Reply:
Right, that's true. Though, this only runs on user input, so I don't really think it matters. But since the readability is the same either way, I changed it.